llvm/tools/llvm-objdump/OffloadDump.cpp

```diff
@@ -48,7 +51,7 @@ static void printBinary(const OffloadBinary &OB, uint64_t Index) {
 }
 
 /// Print the embedded offloading contents of an ObjectFile \p O.
-void llvm::dumpOffloadBinary(const ObjectFile &O) {
+void llvm::dumpOffloadBinary(const ObjectFile &O, std::string ArchName) {
```

Comment:
```suggestion
void llvm::dumpOffloadBinary(const ObjectFile &O, StringRef ArchName) {
```
Unless we really need to have ownership of this string.